app/views/mechanics/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1> Mechanic Show Page </h1>
+
+<p>Name: <%= @mechanic.name %></p>
+<p>years_experience: <%= @mechanic.years_experience %></p>
+
+<h2> Rides: </h2>
+<ul>
+  <% @mechanic.rides.each do |ride| %>
+    <li><%= ride.name %></li>
+  <% end %>
+</ul>
+
+<h2> Add ride to Workload </h2>
+    <%= form_with url: "/mechanics/#{@mechanic.id}/add_ride", method: :patch, local: true do |form| %>
+      <%= label_tag :ride_id, "Ride Id:" %>
+      <%= text_field_tag :ride_id %>
+      <%= submit_tag "Submit" %>
```

Comment:
We don't want to use `form_tag` here. Review the lesson on forms for the correct syntax to use here, `form.label :ride_id`